### ISSUE TITLE
dokku 0.29 compatibility

### ISSUE THREAD

**Issue by aaronstillwell** [aaronstillwell/terraform-provider-dokku#13]
Integration tests are failing on dokku 0.29

https://app.circleci.com/pipelines/github/aaronstillwell/terraform-provider-dokku/133/workflows/b40b6f28-4d20-492e-877b-bab6bc32da94/jobs/402

**Comment by aaronstillwell** [comment]
Looks like the failing tests specific to 0.29 pertain to the setting of domains on apps. Each tests using the app resource seems to fail with something like:

```
=== RUN   TestAccDokkuAppConfigVars
    resource_app_test.go:37: Step 1/1 error: After applying this test step and performing a `terraform refresh`, the plan was not empty.
        stdout
        
        
        Terraform used the selected providers to generate the following execution
        plan. Resource actions are indicated with the following symbols:
          ~ update in-place
        
        Terraform will perform the following actions:
        
          # dokku_app.test will be updated in-place
          ~ resource "dokku_app" "test" {
              ~ domains                 = [
                  - "test-config-vars-vhuxjad8fc.dokku.me",
                ]
                id                      = "test-config-vars-vhuxjad8fc"
                name                    = "test-config-vars-vhuxjad8fc"
                # (5 unchanged attributes hidden)
            }
```

**Comment by aaronstillwell** [comment]
It looks like in 0.29 apps get a default vhost, where as before it used to be blank. I'm guessing `The domains-setup trigger has been removed. Initial app domains will now be automatically setup during app creation.` is in reference to this change as noted [here](https://github.com/dokku/dokku/blob/v0.29.0/docs/appendices/0.29.0-migration-guide.md).

This I guess means the domains attribute needs to be marked as calculated.

**Comment by aaronstillwell** [comment]
This is resolved in the `0.4.3` branch and will close once released pending resolution of the unrelated #14
